### PR TITLE
First Check-In of NNScore Code

### PR DESCRIPTION
**Pull request by rbharath**
This PR contains a completely refactored version of the NNScore codebase, along with test-code and associated test data.

@skearnes This PR is probably too large to meaningfully code-review,  so would you mind just doing a quick once-over instead of a detailed review? This is a first PR, so more changes will follow over the coming weeks.

### CODE REVIEW

File: vs_utils/features/nnscore_pdb.py [hunk 1]
```diff
@@ -0,0 +1,1686 @@
+"""
```
Comment (skearnes) [inline vs_utils/features/nnscore_pdb.py]: This should be in utils.

File: vs_utils/features/nnscore_utils.py [hunk 2]
```diff
@@ -0,0 +1,457 @@
+"""
```
Comment (skearnes) [inline vs_utils/features/nnscore_utils.py]: This should be in utils.

File: vs_utils/features/tests/data/prgr/log.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#################################################################
```
Comment (skearnes) [inline vs_utils/features/tests/data/prgr/log.txt]: Why is this file here?

File: vs_utils/features/tests/data/prgr/run.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+~/autodock/autodock_vina_1_1_2_linux_x86/bin/vina --config conf_active0.txt --log log.txt
```
Comment (skearnes) [inline vs_utils/features/tests/data/prgr/run.sh]: Why is this file here?

File: vs_utils/features/nnscore.py [hunk 5]
```diff
@@ -0,0 +1,735 @@
...
+
+def center(string, length):
+  while len(string) < length:
+    string = " " + string
+    if len(string) < length:
+      string = string + " "
+  return string
+
+
+class binana:
```
Comment (skearnes) [inline vs_utils/features/nnscore.py]: Should this be in a separate package/module? It seems to be a standalone program: http://nbcr.ucsd.edu/data/sw/hosted/binana/

**Comment by skearnes** [comment]
Mostly some organizational comments. It would be helpful if you could put the files that are not part of NNScore in a different PR, like the ones where only the spacing is being modified, so that this PR is self-contained.

**Comment by skearnes** [comment]
Please also add a note to the license file about some parts of the code now being GPL.

File: vs_utils/features/nnscore.py (hunk 5 above)
Comment (skearnes) [inline vs_utils/features/nnscore.py]: Please use uppercase class names: Binana.

File: vs_utils/features/tests/data/prgr/log.txt (hunk 3 above)
Comment (rbharath) [inline vs_utils/features/tests/data/prgr/log.txt]: Sorry, this is cruft. Will get rid of it.

File: vs_utils/features/tests/data/prgr/run.sh (hunk 4 above)
Comment (rbharath) [inline vs_utils/features/tests/data/prgr/run.sh]: Cruft again.

File: vs_utils/features/nnscore.py (hunk 5 above)
Comment (rbharath) [inline vs_utils/features/nnscore.py]: Uppercased it. I'm not sure about whether it should be separate file or not. The author packaged binana within his nnscore script, which is why I've retained the nnscore naming convention. I'm leaning toward leaving it as is unless there's a good for separating it out.

**Comment by rbharath** [comment]
Made requested change:

-) Moved files to utils dir
-) Removed non-nnscore spacing changes from this PR
-) Removed cruft files
-) Updated LICENSE to have GPL text for nnscore files.

PTAL.

**Comment by skearnes** [comment]
LGTM.
